### ISSUE TITLE
Accessibility: Plugin titles

### ISSUE THREAD

**Issue by joedolson** [aspirepress/AspireExplorer#41]
The plugin title in cards has visible text, but the anchor also has `aria-label` and `title`. 

This is just duplication; the title attribute is completely unnecessary if the same text is already visible, and the `aria-label` could potentially mean that if a user auto-translates the page, they will not get the translated version of the title, because the aria-label may be untranslated.

**Comment by joedolson** [comment]
Note: yes, the `aria-label` contains the additional word "details" - in my opinion, this is unnecessary, and should be omitted.
